holochain-overlay/default.nix

```diff
@@ -449,8 +494,18 @@ self: super: {
       windows_x64 = "poEE813SKeRjzo09hFW+BE9JkZzEAMRm2WHG3G7bBYw=";
     }
   );
+  hc_0-4-0 = super.callPackage ./hc/default.nix (
+    self.selectArchConfig {
+      version = "0.4.0"; source = "holochain";
```

Comment:
I think it would be best to reverse the default so that "matthme" is specified on all the legacy binaries instead of adding `source` to all future ones.  However, we can switch that later after this is merged.